diff -p -up ./gnome/notemeister.desktop.tv ./gnome/notemeister.desktop
--- ./gnome/notemeister.desktop.tv	2008-06-17 13:56:19.000000000 +0200
+++ ./gnome/notemeister.desktop	2008-06-17 13:56:22.000000000 +0200
@@ -1,7 +1,6 @@
 [Desktop Entry]
 Name=Notemeister Notes Organizer
 GenericName=Notes Organizer
-Version=0.1
 Comment=Organize small notes in a tree structure
 Exec=notemeister
 Icon=notemeister.svg
@@ -4,7 +4,7 @@ GenericName=Notes Organizer
 Version=0.1
 Comment=Organize small notes in a tree structure
 Exec=notemeister
-Icon=notemeister.svg
+Icon=notemeister
 Terminal=false
 Type=Application
 Categories=Application;Utility;
